scripts/update-cards.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+const { XMLHttpRequest } = require("xmlhttprequest");
+const jsonfile = require('jsonfile');
+
+const latestRequest = new XMLHttpRequest();
+const cardsRequest = new XMLHttpRequest();
+
+let patchNumber = '';
+
+const updateCardsJsonFile = () => {
+    const cardsArray = JSON.parse(cardsRequest.responseText);
+    
+    // Filter for only BG minions
+    const bgCardsArray = []
+    cardsArray.forEach((card) => {
+        if(card["battlegroundsPremiumDbfId"]) {
+            bgCardsArray.push({ id: card["id"], name: card["name"] })
+        }
+    });
+
+    jsonfile.writeFile(
+        "./src/assets/cards.json",
+        bgCardsArray,
+        { spaces: 2 },
+        (err) => err ? console.log(err) : console.log("The cards.json has been updated. 🙌 🎉 🙌 🎉")
+    );  
+}
+
+latestRequest.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+        // Grab the latest patch number from response content.
+        patchNumber = this.responseText.match(/v1\/(\d+)/g)[0].substring(3);
```

Comment:
There's probably a better way to get the latest patch version but.. 🤷‍♂️

Reply:
If it works, that's good enough haha